less/cards.less

```diff
@@ -1,6 +1,16 @@
 .list.advanced-search {
   margin-top: 10px;
 }
+
+.hovered-card() {
+  background-color: #EFEFEF;
+  transition: 0.1s ease-in-out;
+}
+
+.highlght .post {
```

Comment:
there seem to be a typo: `.highlght` > `.highlight`.
Have you checked your modifications?